### PR TITLE
fix(cmake): properly exclude prereleases when fetching latest tag from cmake

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**

/area build

**What this PR does / why we need it**:

Given that we now support prereleases, we do want to exclude them from Falco version computation.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by FedeDP** [comment]
Let's wait for the CI and check the values!
/cc @LucaGuerra

File: cmake/modules/GetGitRevisionDescription.cmake [hunk 1]
```diff
@@ -91,15 +91,16 @@ function(git_get_latest_tag _var)
...
+  # Note how we eclude prereleases tags (the ones with "-alphaX")
   execute_process(COMMAND
           "${GIT_EXECUTABLE}"
           rev-list
+          --exclude "*.*.*-*"
           --tags
           --max-count=1
           WORKING_DIRECTORY
           "${CMAKE_CURRENT_SOURCE_DIR}"
-          COMMAND tail -n1
```
Comment (FedeDP) [inline cmake/modules/GetGitRevisionDescription.cmake:102]: I don't think we ever needed it since we use `max-count=1`; it was probably a leftover.

**Comment by FedeDP** [comment]
> -- Falco version: 0.34.1-200+09c3ad9

(github action build-minimal)

> -- Falco version: 0.34.1-200+09c3ad9

Build-musl

**Comment by FedeDP** [comment]
Uh it seems circleci `build-arm64` and `build-centos7` weren't able to load the Falco version correctly: 
```
Uploading /tmp/packages to /packages
Uploading /tmp/packages/falco-0.0.0-aarch64.deb (27 MB): DONE
Uploading /tmp/packages/falco-0.0.0-aarch64.rpm (22 MB): DONE
Uploading /tmp/packages/falco-0.0.0-aarch64.tar.gz (27 MB): DONE
```

**Comment by FedeDP** [comment]
> Uh it seems circleci build-arm64 and build-centos7 weren't able to load the Falco version correctly:

To fix this, i ported `build-arm64` and `build-centos7` jobs to the same flow we use in gha: load `FALCO_VERSION`  on the host and pass it as a cmake option to the Falco package builder image.

**Comment by FedeDP** [comment]
```
Uploading /tmp/packages to /packages
Uploading /tmp/packages/falco-0.34.1-201+ab56e3f-x86_64.deb (30 MB): DONE
Uploading /tmp/packages/falco-0.34.1-201+ab56e3f-x86_64.rpm (25 MB): DONE
Uploading /tmp/packages/falco-0.34.1-201+ab56e3f-x86_64.tar.gz (30 MB): DONE
```
Fixed build-arm64 and build-centos7. We should be good2go!

**Comment by FedeDP** [comment]
> E: Failed to fetch http://ports.ubuntu.com/ubuntu-ports/pool/main/g/gnupg2/gnupg-utils_2.2.4-1ubuntu1.6_arm64.deb  Unable to connect to ports.ubuntu.com:http: [IP: 185.125.190.39 80]

Arm64 is having some connection issues today; i already tried to restart it :/

**Comment by LucaGuerra** [comment]
It looks great to me! Thank you for editing that tricky cmake function!

**Comment by FedeDP** [comment]
Still failing :/

**Comment by FedeDP** [comment]
:green_circle: finally!

**Review by leogr — APPROVED** [review]

**Review by Andreagit97 — APPROVED** [review]
/approve

**Pull request merged**